client/store/admin.js

```diff
@@ -55,6 +55,8 @@ export const fetchSingleProduct = (id) => async (dispatch) => {
 
 export const updateProduct = (product, history) => async (dispatch) => {
   try {
+    // JOE_CR: There is a lot of repeated utility logic of accessing, checking,
+    // and incorporating the token for the request. How can you centralize this?
```

Comment:
I moved the token logic to a new file called headers. Now, we just import the headers file, and use the variable to handle this. It looks much cleaner.